--- a/src/spatialaudio/CMakeLists.txt
+++ b/src/spatialaudio/CMakeLists.txt
@@ -11,7 +11,7 @@ qt_internal_add_module(SpatialAudio
         "../3rdparty/resonance-audio/resonance_audio"
         "../3rdparty/resonance-audio"
         "../resonance-audio"
-        "../3rdparty/eigen"
+        "/usr/include/eigen3"
     LIBRARIES
         Qt::MultimediaPrivate
         Qt::BundledResonanceAudio
--- a/src/resonance-audio/CMakeLists.txt
+++ b/src/resonance-audio/CMakeLists.txt
@@ -204,7 +204,7 @@ qt_internal_add_3rdparty_library(Bundled
         ${RA_SOURCE_DIR}
         ${PFFFT_DIR}
         ${SADIE_HRTFS_DIR}
-        ../3rdparty/eigen
+        /usr/include/eigen3
 )

 # Required by pffft on certain PowerPC archs
